### PR TITLE
Changed twitter link to x

### LINKED ISSUE #3994: Twitter link should be changed to x links
In `setup.py`'s `project_urls` metadata, change Twitter link from `https://twitter.com/plone` to `https://x.com/plone`.

### PR DESCRIPTION
**Pull request by SarthakPaandey**
As part of the recent rebranding of Twitter to X, we propose updating all references to Twitter links (twitter.com) in the codebase and documentation to x.com. This change aligns with the platform's new branding and ensures consistency in all related references.
This update is necessary to maintain brand consistency within the project and provide an accurate reflection of the external service's current branding. By updating the links to x.com, we ensure that users and developers are not confused by outdated references to Twitter.
Fix #3994

### CODE REVIEW

**Comment by SarthakPaandey** [comment]
@jenkins-plone-org please run jobs

**Comment by SarthakPaandey** [comment]
I have signed it.

**Comment by SarthakPaandey** [comment]
@gforcada kindly look into this pull request.

**Comment by rohnsha0** [comment]
I'm not sure we need this PR. As far as I know, any attempt to access twitter.com gets redirected to x.com. That means all the links that are currently pointing to twitter.com aren't broken.

Am I missing something? @SarthakPaandey 
&nbsp;

> @gforcada kindly look into this pull request.

please dont tag like this. request a review instead

**Review by rohnsha0 — COMMENTED** [review]
I'm not sure we need this PR. As far as I know, any attempt to access twitter.com gets redirected to x.com. That means all the links that are currently pointing to twitter.com aren't broken.

Am I missing something? @SarthakPaandey 
Infact, changing the user-side mentions of `Twitter` to `X` makes more sense
&nbsp;

> @gforcada kindly look into this pull request.

please dont tag like this. request a review instead

**Comment by SarthakPaandey** [comment]
I know but for future reference it is kind of good and also I want to contribute something to the Plone community.

**Comment by SarthakPaandey** [comment]
Yes @rohnsha0.

**Comment by rohnsha0** [comment]
Okay, but please change the references in https://github.com/plone/plone.base/blob/3a5aef0e158da872d2280cb0150d94c65417120f/src/plone/base/interfaces/controlpanel.py#L1593-L1602 as well. Submit a separate PR for those changes, and include a reference to this PR.

**Review by rohnsha0 — CHANGES_REQUESTED** [review]
Ill suggest changing to

```
project_urls={
        # other entries...
        "X (formerly Twitter)": "https://x.com/plone",
        # other entries...
    },
```

**Comment by SarthakPaandey** [comment]
> Ill suggest changing to
> 
> ```
> project_urls={
>         # other entries...
>         "X (formerly Twitter)": "https://twitter.com/plone",
>         # other entries...
>     },
> ```

should i add x.com or not?

**Comment by rohnsha0** [comment]
> > Ill suggest changing to
> > ```
> > project_urls={
> >         # other entries...
> >         "X (formerly Twitter)": "https://twitter.com/plone",
> >         # other entries...
> >     },
> > ```
> 
> should i add x.com or not?

Ofc

**Comment by stevepiercy** [comment]
@SarthakPaandey you should first satisfy contributing requirements, as I [mentioned in the original issue](https://github.com/plone/Products.CMFPlone/issues/3994#issuecomment-2269758616). We cannot accept a PR until you do so. Thank you!

**Comment by SarthakPaandey** [comment]
I have already signed the agreement. Do i need to submit it anywhere?

**Comment by stevepiercy** [comment]
> I have already signed the agreement. Do i need to submit it anywhere?

Yes. Again, please read my original comment about contributing. Thank you!

**Comment by mauritsvanrees** [comment]
I as Plone Release Manager am happy to accept this small change, even if the contributor agreement has not been handled yet.  I can make such exceptions.
First changing the title to "X (formerly Twitter)" as suggested may be good.

**Comment by petschki** [comment]
I'd remove it completely and additionally delete the plone account on X ... we really should not support this platform.

**Comment by rohnsha0** [comment]
> I'd remove it completely and additionally delete the plone account on X ... we really should not support this platform.

As of today, Plone is available on X... Right?!

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "Forum": "https://community.plone.org/",
         "Chat": "https://discord.gg/zFY3EBbjaj",
         "Mastodon": "https://plone.social/@plone",
-        "Twitter": "https://twitter.com/plone",
+        "Twitter": "https://x.com/plone",
```
Comment (stevepiercy) [inline setup.py:44]:
```suggestion
        "X (formerly Twitter)": "https://x.com/plone",
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by stevepiercy** [comment]
> I'd remove it completely and additionally delete the plone account on X ... we really should not support this platform.

I wholeheartedly agree. That discussion, however, needs to take place amongst the @plone/marketing-communications Team, and is outside the scope of this PR.

Meanwhile each of us can unfollow Plone on Xwitter, and follow on [Mastodon](https://plone.social/@plone), bidding them a fond "Hello there!".

@mauritsvanrees can we just merge this PR for now?

**Comment by SarthakPaandey** [comment]
image
I have signed it already.

> @SarthakPaandey you need to sign the Plone Contributor Agreement to merge this pull request.
> 
> Learn about the Plone Contributor Agreement: https://plone.org/foundation/contributors-agreement

**Comment by stevepiercy** [comment]
@SarthakPaandey would you please follow up per https://plone.org/foundation/contributors-agreement:

> ## I signed the Agreement, but haven't been given access yet?
>
> Please allow a week for a response. Volunteers process the Contributor Agreements. If you have issues, such as making a spelling error in your email or GitHub username, please contact agreements@plone.org.

**Review by mauritsvanrees — APPROVED** [review]
This change is fine even if something is wrong with regards to the contributor agreement.  I merge.

Thanks!

**Pull request merged**